packages/opencensus-exporter-stackdriver/test/test-stackdriver-monitoring.ts

```diff
@@ -198,71 +185,178 @@ describe('Stackdriver Stats Exporter', function() {
 
   /* Should create a Stackdriver Metric Descriptor */
   describe('Create Stackdriver Metric Descriptors', () => {
-    testViews.map((view) => {
-      it(`Should create a ${view.description} Metric Descriptor`, async () => {
-        if (dryrun) {
-          nocks.metricDescriptors(PROJECT_ID, null, null, false);
-        }
-        await exporter.onRegisterView(view).then(() => {
-          return assertMetricDescriptor(
-              exporterTestLogger.debugBuffer[0], view);
-        });
-      });
-    });
+    it(`Should create a MeasureDouble with Metric Kind Gauge Metric Descriptor`,
+       async () => {
+         if (dryrun) {
+           nocks.metricDescriptors(PROJECT_ID, null, null, false);
+         }
+         await exporter.onRegisterView(lastValueDoubleView).then(() => {
+           return assertMetricDescriptor(
```

Comment:
As we discussed/agreed before, please use regular assert in test instead of calling method. The same for lines below in this PR